src/generate-names.h

```diff
@@ -23,7 +23,16 @@ namespace wabt {
 
 struct Module;
 
-Result GenerateNames(struct Module*);
+Result GenerateNames(struct Module*, bool numbered_names, bool dollar_prefix);
```

Comment:
Default args maybe?

Also I'm never a fan of `foo(bar, true, true)` at the callsite. So,

```cpp
enum NameOptions {
  nameNumbered = 1 << 0,
  nameDollared = 1 << 1,
};

// ...

Result GenerateNames(struct Module*, NameOptions options = nameNumbered | nameDollared);
```

Reply:
Done